argocd/eoepca/mlops/parts/app-sharinghub.yaml

```diff
@@ -131,7 +131,7 @@ spec:
           repository: eoepca/sharinghub
           pullPolicy: Always
           # Overrides the image tag whose default is the chart appVersion.
-          tag: "latest"
+          tag: "0.3.0"
```

Comment:
Same comment as above

Reply:
I updated the charts to use "0.3.0" by default.